tox.ini

```diff
@@ -184,15 +184,18 @@ deps =
 whitelist_externals =
     {[testenv]whitelist_externals}
     sphinx-build
-commands = sphinx-build -E -W -b html -d build/doctrees source build/html
+
 
 [testenv:docs]
 sitepackages = {[docsbase]sitepackages}
 changedir = {[docsbase]changedir}
 install_command = {[docsbase]install_command}
 deps = {[docsbase]deps}
 whitelist_externals = {[docsbase]whitelist_externals}
-commands = sphinx-build -v -E -T -b html -d build/doctrees source build/html
+commands =
+    sphinx-apidoc --force -o source/modules/ ../deluge ../deluge/plugins
+    sphinx-build -v -E -T -b html -d build/doctrees source build/html
```

Comment:
That seems a waste...

How about print out those two lines and call `BuildDocs`?